apps/backend/src/security/security.service.ts

```
@@ -218,21 +230,14 @@ export class SecurityService {
 			// Remove non-ASCII characters (keep only characters 0-127)
-			let asciiOnly = ''
-			for (let i = 0; i < sanitized.length; i++) {
-				const char = sanitized.charAt(i)
-				const code = char.charCodeAt(0)
-				if (code >= 0 && code <= 127) {
-					asciiOnly += char
-				}
-			}
-			sanitized = asciiOnly
+			// eslint-disable-next-line no-control-regex
+			sanitized = sanitized.replace(/[^\x00-\x7F]/g, '')
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)